pep-9999.rst

```diff
@@ -149,6 +149,47 @@ Two kinds of structured comments are accepted:
 * A ``# type: ignore`` comment at the end of any line, which suppresses all type
   errors in that line.
 
+Version and Platform Checks
+---------------------------
+
+Type stubs for libraries that support multiple Python versions can use version
+checks to supply version-specific type hints. Type stubs for different Python
+versions should still conform to the most recent Python version's syntax, as
+explain in the Syntax_ section above.
+
+Version checks are if-statements that use ``sys.version_info`` to determine the
+current Python version. Version checks should only check against the ``major`` and
+``minor`` parts of ``sys.version_info``.
+
+Stub files support various forms of version checks::
+
+    if sys.version_info >= (3,):
+        # Python 3-specific type hints. This tuple-based syntax is recommended.
+    else:
+        # Python 2-specific type hints.
+
+    if sys.version_info.major >= 3:
```

Comment:
We don't want to encourage this syntax, so reword this.

Reply:
from the meeting, stubs should not support: version_info.major, .startswith and `in` checks